mikupad.html

```diff
@@ -3829,8 +3865,21 @@
 		showProbs && promptChunks[currentPromptChunk?.index]?.completion_probabilities?.[0]?.probs,
 		[promptChunks, currentPromptChunk, showProbs]);
 
+	const sidebar = useRef(null);
+	const [height, setHeight] = useState(0);
+	const [isMobile, setIsMobile] = useState(false);
+	useEffect(() => {
+		setHeight(sidebar.current.scrollHeight);
+		const observer = new SVResizeObserver(() => {
+			setIsMobile(window.innerWidth < 767.8);
+			setHeight(sidebar.current.scrollHeight);
+		});
+		observer.observe(sidebar.current);
+		return () => observer.disconnect();
+	});

```

Comment:
you need to pass an empty array as the second parameter otherwise the useEffect hook [runs every render](https://stackoverflow.com/a/58579462)!
```suggestion
	}, []);
```